docs/codeReview.md

```diff
@@ -12,9 +12,16 @@ A PR advances from self-review to reviewer-review and finally maintainer review
 - It should be a reviewer’s goal to do the code review within 1 business day. If you’re unable to do it, leave a comment.
 - If you’re busy it’s ok to assign someone else as a reviewer.
 - When submitting a PR, provide context. What's the task? What's changed? Why? If you can, record a Loom to share why things are done that way. It'll be easier for a reviewer to understand what decision you've made and why.
-- When reviewing a PR and requesting changes, be mindful that the PR author won't always have the right background to understand what are you requesting. Make your comments meaningful, and record a Loom if needed.
+- When reviewing a PR and requesting changes, be mindful that the PR author won't always have the right background to understand what are you requesting. Make your comments meaningful, and record a Loom if needed. 
 - [The right balance](https://docs.gitlab.com/ee/development/code_review.html#the-right-balance)
 
+## Sprint Board
+
+- Issues can go in these columns: To Prioritize, Backlog, To Do, In Progress, Stuck.
```

Comment:
what happens to the issue when the PR is submitted? done?

Reply:
Answered in 3rd bullet point:

> These issues shall stay "In Progress" and when the PR gets merged, they'll be moved to "Done" automatically.

I didn't include "Done" as a column in these first 2 bullet points because nothing should get moved to "Done" manually, but not sure how to state that clearly

Reply:
oh, how did I miss that? thanks!